src/pages/calendar-dropdown/calendar-dropdown.ts

```diff
@@ -29,7 +29,6 @@ export class CalendarDropdownPage {
 @Component({
```

Comment:
Remove this entire component. It doesn't do anything. (see the button text "Learn Ionic" and "Documentation", etc.). Those are incorrect for this app.

Reply:
I guess I’m confused, I already removed the @Component -class myPage stuff. I’m probably looking at the wrong thing but if I remove anything rise from the page, I’ll get an error. Do you mean the import from ionic-angular stuff? That’s what the node.js command window is saying is unused.

Reply:
Did you mean this code in the calendar.ts file? I removed that as a test on my local version and things were still fine.

@Component({
	 template: `
    <ion-list>
      <button ion-item (click)="close()">Learn Ionic</button>
      <button ion-item (click)="close()">Documentation</button>
      <button ion-item (click)="close()">Showcase</button>
      <button ion-item (click)="close()">GitHub Repo</button>
    </ion-list>
   `
})